src/StorageSync/StorageSync/Interop/DataObjects/AutoUpdatePolicy.cs

```diff
@@ -0,0 +1,26 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="AutoUpdatePolicy.cs" company="Microsoft Corporation.">
+//   All rights reserved.
+// </copyright>
+// <summary>
+//   Auto-update policy detail
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Commands.StorageSync.Interop.DataObjects
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    [StructLayout(LayoutKind.Sequential)]
+    public struct AutoUpdatePolicy
+    {
+        [MarshalAs(UnmanagedType.BStr)]
```

Comment:
more code that I don't understand how is related to MI or SEP Provisioning Status

Reply:
we port all Interop library